ext/RMagick/rmpixel.c

```diff
@@ -770,13 +827,24 @@ Pixel_initialize(int argc, VALUE *argv, VALUE self)
 
     Data_Get_Struct(self, Pixel, pixel);
 
+#if defined(IMAGEMAGICK_7)
+    pixel->alpha = OpaqueAlpha;
+#endif
+
     switch(argc)
     {
         case 4:
+#if defined(IMAGEMAGICK_7)
+            if (argv[3] != Qnil)
+            {
+                pixel->alpha = APP2QUANTUM(argv[3]);
```

Comment:
Looking at the definition for `APP2QUANTUM`, it is just `rm_app2quantum((n)`. Why would that have a `define` instead of calling the function? Is it to be consistent with the other `define`s?

Reply:
Yeah we could use call the function here. Something for a new PR? But it would be nice if we could wait with these after the IM7 merges because it will probably break my changes.